src/content/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-apm-agents.mdx

```diff
@@ -21,158 +21,1190 @@ redirects:
+4. View your logs within the context of your apps or infrastructure in [New Relic One](/docs/logs/log-management/ui-data/use-logs-ui/#links).
+
+The main differences in this procedure are which log forwarders are available for your agent, and how to configure the specific log forwarder.
+
+## C SDK [#enable-c-sdk]
+
+Logs in context is not available for APM apps monitored by the C SDK. Instead, you can build your own logging extension library by using the [Log API](/docs/logs/log-management/log-api/introduction-log-api/). Also make sure that [distributed tracing](/docs/distributed-tracing/enable-configure/language-agents-enable-distributed-tracing/) is enabled.
+
+## Go [#enable-go]
+
+To enable logs in context for APM apps monitored by Go:
+
+1. Make sure your Go agent supports logs in context (agent version 2.12 or higher) and has [distributed tracing enabled](/docs/distributed-tracing/enable-configure/language-agents-enable-distributed-tracing/).
+2. Install the [Logrus logging framework version 1.4.0 or higher](https://github.com/sirupsen/logrus) to forward logs to New Relic.
+3. Configure logs in context for Go using the Logrus extension.
+
+<CollapserGroup>
+  <Collapser
+    id="go-configure"
+    title="Go configuration with Logrus"
+  >
+
+1. Import the following packages in files where Logrus is configured:
+
+   ```
+   import (
+       log "github.com/sirupsen/logrus"
+       "github.com/newrelic/go-agent/v3/integrations/logcontext/nrlogrusplugin"
+       "github.com/newrelic/go-agent/v3/newrelic"
+   )
+   ```
+2. Set the Logrus formatter to `nrlogrusplugin.ContextFormatter`:
+
+   ```
+   logger := log.New()
+   logger.SetFormatter(nrlogrusplugin.ContextFormatter{})
+   ```
+
+   OR
+
+   If you are using the Logrus standard logger:
+
+   ```
+   log.SetFormatter(nrlogrusplugin.ContextFormatter{})
+   ```
+
+   The logger will now look for a `newrelic.Transaction` inside its context and decorate logs accordingly.
+3. Update your standard logging call to include contextual logging, and pass this to the logger.
+
+   For example, instead of `logger.Info("Hello New Relic!")`, use
+
+   ```
+   ctx := newrelic.NewContext(context.Background(), txn)
+   logger.WithContext(ctx).Info("Hello New Relic!")
+   ```
+
+   where `txn` is the current running transaction.
+
+  </Collapser>
+
+</CollapserGroup>
+
+4. To verify that you have configured the log forwarder extension correctly, run your application and verify that the logging you have configured contains the following:
+
+   * Includes `trace.id` and `span.id` fields
+   * Has properly-formatted JSON lines
+
+If everything is configured correctly and your data is being reported, you should see data in the [New Relic Logs UI](/docs/logs/log-management/ui-data/use-logs-ui/) using the query operator `has:span.id has:trace.id`.
```

Comment:
> and your data is being reported

Here it might be worth using the word "forwarded", since it'll rely on a log forwarder to be sent to New Relic.
Perhaps we should link to the "setting up a log forwarder" docs from here, too?